src/uwtools/drivers/cdeps.py

```diff
@@ -127,3 +128,6 @@ def _model_stream_file(self, group: str, path: Path, template_file: str) -> None
             output_file=path,
             values_src=self._driver_config[group],
         )
+
+
+set_driver_docstring(CDEPS)
```

Comment:
This call attaches parameter descriptions from the parent class to the driver class' docstring. The pattern is repeated in each driver module.

Before:
<img width="1026" alt="image" src="https://github.com/user-attachments/assets/0bb6b762-ea81-46aa-9b45-102b3bdbb427">
After:
<img width="766" alt="image" src="https://github.com/user-attachments/assets/a765938a-d34b-4592-9e60-f1619052647d">